com.unity.render-pipelines.universal/Documentation~/writing-shaders-urp-unlit-texture.md

```diff
@@ -0,0 +1,143 @@
+# Drawing a texture
+
+The shader in this example draws a texture on the mesh.
+
+Use the ShaderLab code from section [URP unlit shader with color input](writing-shaders-urp-unlit-color.md) and make the following changes to it:
+
+1. In the Properties block, replace the existing code with the `_BaseMap` property definition.
+
+    ```c++
+    Properties
+    { 
+        _BaseMap("Base Map", 2D) = "white"
+    }
+    ```
+
+    When you declare a texture property in the Properties block, Unity adds the `_BaseMap` property with the label __Base Map__ to the Material, and adds the Tiling and the Offset controls.
+
+    !["Texture property with Tiling and Offset controls"](Images/shader-examples/unlit-shader-tutorial-texture-property-in-inspector.png)
+    
+    The `_BaseMap` property name is a reserved name. When you declare a property with this name, Unity uses this property as the [main texture](https://docs.unity3d.com/ScriptReference/Material-mainTexture.html) of the Material. 
+
+2. In `struct Attributes` and `struct Varyings`, add the `uv` variable for the UV coordinates on the texture:
```

Comment:
Would details about users being able to have many TEXCOORDX fit in here??

Reply:
Thanks!
Created ticket: https://unity3d.atlassian.net/browse/DOC-2085